configfs/fakertmr/fakertmr.go

```diff
@@ -193,13 +193,19 @@ func (r *RtmrSubsystem) WriteFile(name string, content []byte) error {
 	return r.WriteAttr(path.Join(r.Path, p.Entry), p.Attribute, content, r.rtmrIndexMap)
 }
 
+// Close removes the temporary files in the fake rtmr subsystem.
+func (r *RtmrSubsystem) Close() error {
+	return os.RemoveAll(r.Path)
+}
+
 // CreateRtmrSubsystem creates a new rtmr subsystem.
 // The current subsystem only supports TDX.
 func CreateRtmrSubsystem() *RtmrSubsystem {
```

Comment:
I would recommend instead to take the root directory as an argument, and pass in t.TempDir() so you don't need to Close() after the test completes. That's already part of the testing library.

Reply:
Thanks for the suggestion. Done